README.md

```diff
@@ -70,21 +69,106 @@ of 30 mins of learning "***web super powers***",
+
+![image](https://cloud.githubusercontent.com/assets/16775804/24111767/6f6f2512-0d8f-11e7-93db-c6976bfb8d9e.png)
+
+#### Let's get started
+
+- **Pick a user journey that you want to test.** Start small. We'll start with the following journey:
+
+  **From this README > click the giphy link in the tutorial description at the top > arrive at a gif with the hashtag 'nightmare'**
+
+  This is what it will look like:
+
+  ![demo](https://cloud.githubusercontent.com/assets/16775804/24203856/be1985e2-0f0e-11e7-8ff3-ab62df8f8cc9.gif)
+
+- Before starting the recording make sure your browser is on the page you wish to start on. In our case it's this README.
+
+- To start the test click the camera symbol. It will turn green to show it is recording.
+
+![image](https://cloud.githubusercontent.com/assets/16775804/24113367/7df1feca-0d94-11e7-99f4-26775eb9f2ee.png)
+
+- The first action you need the script to record is the url of the page you are on. To do this refresh your page (it will appear as `.goto(www....)` in your finished script).
+
+- Next navigate as you normally would, following the path that you want the test to take. So for this test you would click on the giphy link in the tutorial description.
+
+![image](https://cloud.githubusercontent.com/assets/16775804/24113044/57cb6eb2-0d93-11e7-8973-bc82cfbe7519.png)
+
+- This test is only a short one and so having clicked on the giphy link and having arrived on the giphy #nightmare page we are ready to end the recording.
+To do this, stop the recording by clicking on the green camera symbol. It will turn black to show recording has ended. You will see a blue notification which indicates that you've recorded a script:
+
+<div style="text-align:center"><img src ="https://cloud.githubusercontent.com/assets/16775804/24113514/f279fdf6-0d94-11e7-82d9-2d55e40c8405.png" /></div>
+
+- Click this to see the recorded script (the bit of code you'll need to use in your project for the test). Copy and paste this code into a new file in your project.
+
+### Turning the Daydream script into a Nightmare test
+
+Daydream does the hard work of recording our actions into a Nightmare test. However, we still need to add to the code it gives us for our test to work. Let's understand what Daydream has done for us:
+
+**1.** Set up your code to run Nightmare tests
+
+**2.** Written the code that tracks the user's journey throughout the code (ie. going to a given url, clicking on a button)
+
+<div style="text-align:center"><img src ="https://cloud.githubusercontent.com/assets/16775804/24255398/4d9567a8-0fdd-11e7-9888-0d33f330adf0.png" height="300px"/></div>
+
+If you're non-technical try making sense of the commands in part **2** (on the screenshot). They represent common user actions such as clicking on a page or taking a screenshot. The text within the parentheses indicates what these commands apply to ie. what html/css element is being clicked on.
```

Comment:
note: we tend to use "semantic" line breaks to avoid having super-long-lines of text.